modules/subscriptions/views.php

```diff
@@ -6,6 +6,19 @@ class Jetpack_Subscriptions_Widget extends WP_Widget {
 	 * @var array When printing the submit button, what tags are allowed
 	 */
 	static $allowed_html_tags_for_submit_button = array( 'br' => array() );
+	/**
+	 * Use this variable when printing the message after submitting an email in subscription widgets
+	 *
+	 * @var array what tags are allowed
+	 */
+	public static $allowed_html_tags_for_message = array(
+		'a'  => array(
+			'href'  => array(),
+			'title' => array(),
+			'rel'   => array(),
```

Comment:
You'll want to add `target` in there as well.

```suggestion
			'rel'    => array(),
			'target' => array(),
```